apps/comps/components/clock/index.tsx

```diff
@@ -34,14 +59,15 @@ export const CountdownClock: React.FC<CountdownClockProps> = ({
       } else {
         setTimeLeft({ hours: 0, minutes: 0, seconds: 0 });
         setIsCountdownFinished(true);
+        setDuration("");
```

Comment:
same thing here...maybe "TBA"? (this might be related to the points below.)

Reply:
Related to the above response, when the countdownFinished value is `true` the duration and clock are hidden.
I was following the existing impl here by setting the time and duration back to their initial values.